changes

```diff
@@ -1,48 +1,60 @@
-indexForm.submit.send();
-Line 741 add:
-this.preSendSet();
-this.store();
+Dan:
+
+Kev:
+
+Rob:
+
+--index.js--
+
+// Consider adding failure to the Age varable if the DOB is updated to be out of range i.e. if a user initially adds an acceptable DOB then changes it,
+// currently the Age variable remains updated with previous number - may be unnecesary though
+line 107 add: 					indexForm.inputs.age.set(-1);
+line 104 move to 109:					//indexForm.inputs.age.blur();
+
+// If permitting input of previous dates for Protocol start time, your calculation of age in months and all contingent could be wrong
+// e.g. line 338 which seems to calculate age in months c.f. current time, not age at protocol start
+// add a time check for protocols generated far in the past by accident? Not in index.js as I can see, can even be before they were born
+// Such as:
+line 184:   				if (this.goodFormat() && this.notFuture() && this.isRecent()) return true;
+insert at line 195:
+isRecent: function(){
+	if (moment().diff(moment(this.value(), "DD/MM/YYYY HH:mm"), 'days') < 32) return true;
+	return false;
+}
+insert at line 198
+if (!this.isRecent()) return "Protocol start date/time must be within the last month.";
+
+// I wonder if you want to keep the error message up for weight even when override is ticked, though could make it less prominent
+// Should there be a hard limit that cannot be overriden, like 150kg max?
+
+// Line 391 & 396 - should this refer to the upper and lower limits within calcVars.js not be hard coded?
+
+// Line 703 - On testing I am not getting this box to confirm I wish to proceed with the 200kg Joe Bloggs - fixed
+// Still with disabled popup blocker. I can run the function from the console and the popup comes.
+
+// Updates we did together to make the override box pop up:
+Line 681:       		    	
+if (indexForm.inputs.weight.isOverridden()) {
+    this.confirmOverride();
+} else {
+    this.send();
+}
+Line 721:
+indexForm.submit.send();
+Line 741 add:
+this.preSendSet();
+this.store();
+
+// You don't have any input screening for variables as tested with bizarre weights (e.g. 12.345678901234567890) - these slightly break your formatting, and carrying of digets in the calculations
+// Should weight be rounded toFixed(1) and pH toFixed(2) perhaps at the input and storage stage?
+
+--calcVars.js--
+
+// Minor changes to add units to limit comments
+Line 86: Changed to volumeToRate(volume, unitTime) and be generic for whatever input
+
+--showWorking.js--
+
+// I note this is where you round to 1 or 2 decimal places for rates - might it be neater to move this within the calcVars functions? Perhaps a reason for this I am not aware of but it seems to de-couple a key step from showWorking and the actual calcs going into the document
+// Minor changes to harmonise square bracket location around variables
+// The divide sign ony my screen looks very similar to + would "/" be understandable, or perhaps all the maths symbols could be made larger?
```

Comment:
You could try replacing:
 &#247; (the HTML entity for divide symbol)
with:
<span style="font-size:2em;">&#247;</span>
the 2em will increase the size of the divide symbol by a factor of 2 - change the value as required. If you still don't think it looks ok then by all means change to a /.